### PR TITLE
Changing expected model format.

### PR DESCRIPTION
**Pull request by ptpham**

### CODE REVIEW

**Comment by ptpham** [comment]
Hold on. I haven't fixed the  ... tests ... T__T

**Comment by ptpham** [comment]
Requires https://github.com/kokorigami/kokorigami/pull/59

**Comment by ptpham** [comment]
Also, don't bother refining when you check in the inspector anymore, I don't think we'll need it. XD

**Comment by ptpham** [comment]
Quick reminder, you will need to provide a thickness value to see the goodness. I prefer 0.01.

File: src/model.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+function computeFrameNormals(frame) {
+  var points = frame.points;
+  var normals = [];
+
+  for (var i = 0; i < points.length; i += 3) {
+    var normal = getPlaneNormal([], points[i], points[i+1], points[i+2]);
+    normals[i] = normals[i+1] = normals[i+2] = normal;
+  }
+
+  frame.normals = normals;
```
Comment (cixzhang) [inline src/model.js]: Could we have this function just return the normals? :D

**Review by cixzhang — APPROVED** [review]
Looks good! I just have one stylistic request :D

**Pull request merged**